test/storage/file.js

```diff
@@ -485,6 +485,23 @@ describe('File', function() {
       file.metadata = metadata;
       file.createReadStream({ start: startOffset, end: endOffset });
     });
+
+    it('should accept range start and end as 0', function(done) {
+      var startOffset = 0;
+      var endOffset = 0;
+
+      request_Override = function(opts) {
+        setImmediate(function () {
+          var expectedRange = 'bytes=0-0';
```

Comment:
Google Storage ranges [are inclusive](https://cloud.google.com/storage/docs/reference-headers#range), so `range=0-999` is the first 1000 bytes. `range=0-0` would be the first byte.